### PR TITLE
Update da.po 20200422

### LINKED ISSUE #706: Update translation of message strings into Danish
Before the release of v2019.2 the translation into French, Swedish and Danish must be updated. For all three languages there are message strings without translation. This issue is for translation into Danish.

Relates to #696 and #705.

@haarbo, is it possible for you to help us with this? Please see updated instructions in "[Translation]", section "For translators".

[Translation]: https://github.com/zonemaster/zonemaster-engine/blob/develop/docs/Translation.pod

### PR DESCRIPTION
**Pull request by haarbo**
Another go at fixing missing translations

Resolves #706, replaces #711.

### CODE REVIEW

**Comment by matsduf** [comment]
@haarbo, does this PR replace PR #711? Or is this in addition to the other PR?

**Comment by matsduf** [comment]
@mattias-p, do you have any comments on this?

File: share/da.po [hunk 1]
```diff
@@ -1761,6 +1913,73 @@ msgstr ""
...
+"{ns}/{address} returned no DS record created by algorithm {algorithm_number} "
+"({algorithm_mnemonic}) for zone {zone}, which is required."
+msgstr ""
+"{ns}/{address} returnerede ingen DS-record oprettet af algoritme {algorithm_number} "
+"({algorithm_mnemonic}) for zonen {zone}, hvilket er krævet."
+
+#. DNSSEC:NO_RESPONSE_DS
+#, perl-brace-format
+msgid "{ns}/{address} returned no DS records for {zone}."
+msgstr "{from} returnerede ingen DS-records for {zone}."
```
Comment (mattias-p) [inline share/da.po:1981]: "{from}" -> "{ns}/{address}"

File: share/da.po [hunk 2]
```diff
@@ -265,6 +281,39 @@ msgstr ""
...
+#. DNSSEC:DNSKEY_TOO_SMALL_FOR_ALGO
+#, perl-brace-format
+msgid ""
+"DNSKEY with tag {keytag} and using algorithm {algorithm_number} "
+"({algorithm_description}) has a size ({keysize}) smaller than the minimum "
+"one ({keysizemin})."
+msgstr ""
+"DNSKEY med tag {keytag} bruger en algoritme {algorithm_number} "
+"({algorithm_description}) med størrelsen ({keysize}), der er mindre end den "
+"mindst tilladte værdi ({keysizemax})."
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/da.po]: Fixed.

File: share/da.po [hunk 3]
```diff
@@ -325,34 +363,38 @@ msgstr "Delegeringen fra forælder- til børnezonen er korrekt signeret."
...
 #, perl-brace-format
 msgid ""
-"Delegation lists enough ({count}) nameservers that resolve to IPv4 addresses "
-"({addrs}). Lower limit set to {minimum}."
+"Delegation lists enough ({count}) nameservers ({nss}) that resolve to IPv4 "
+"addresses ({addrs}). Lower limit set to {minimum}."
 msgstr ""
-"Delegeringen indeholder et tilstrækkeligt antal ({count}) navneservere med IPv4-adresser. Mindsteværdien er {minimum}."
+"Delegeringen indeholder et tilstrækkeligt antal ({count}) navneservere ({nss}) med "
+"IPv4-adresser. Mindsteværdien er {minimum}."
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/da.po]: Fixed.

File: share/da.po [hunk 4]
```diff
@@ -325,34 +363,38 @@ msgstr "Delegeringen fra forælder- til børnezonen er korrekt signeret."
...
 #, perl-brace-format
 msgid ""
-"Delegation lists enough ({count}) nameservers that resolve to IPv6 addresses "
-"({addrs}). Lower limit set to {minimum}."
+"Delegation lists enough ({count}) nameservers ({nss}) that resolve to IPv6 "
+"addresses ({addrs}). Lower limit set to {minimum}."
 msgstr ""
-"Delegeringen indeholder et tilstrækkeligt antal ({count}) navneservere med IPv6-adresser. Mindsteværdien er {minimum}."
+"Delegeringen indeholder et tilstrækkeligt antal ({count}) navneservere ({nss}) med "
+"IPv6-adresser. Mindsteværdien er {minimum}."
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/da.po]: Fixed.

File: share/da.po [hunk 5]
```diff
@@ -787,11 +830,43 @@ msgstr "Navneserveren {ns}/{address} understøtter ikke EDNS0 (svarer FORMERR)."
...
+#, perl-brace-format
+msgid ""
+"Nameserver {ns}/{address} for zone {zone} responds with both NSEC and NSEC3 "
+"records when only one record type is expected."
+msgstr ""
+"Navneserveren {ns}/{address} for zonen {zone} svarede både NSEC- og "
+"NSEC3-records på trods af, at kun en recordtype er forventet i svaret."
+
+"Navneserveren {ns}/{address} for zonen {zone} returnerede flere ({count}) SOA records på SOA "
+"forespørgsler."
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/da.po]: Fixed.

File: share/da.po [hunk 6]
```diff
@@ -839,19 +916,58 @@ msgstr ""
...
+msgstr ""
+"Navneserver {ns}/{address} returnerede en RRSIG som ikke kan verificeres "
+"med den tilhørende DNSKEY (med keytag {keytag})"
+
+#. DNSSEC:RRSIG_NOT_MATCH_DNSKEY
+#, perl-brace-format
+msgid ""
+"Nameserver {ns}/{address} responded with an RRSIG with unknown keytag "
+"{keytag}."
+msgstr "Navneserveren {ns}/{address} returnerede en RRSIG med et ukendt keytag."
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/da.po]: Fixed.

File: share/da.po (hunk 1 above)
Comment (matsduf) [reply share/da.po]: Fixed.

**Comment by matsduf** [comment]
@haarbo, I pushed a commit that fixed the glitches. @mattias-p, can you re-review?

**Pull request merged**